sec-keys/openpgp-keys-eduvpn/files/227FF3F8F829D9A9314D9EBA02BB8048BBFF222C.asc

```diff
@@ -0,0 +1,10 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
@ulm Is this OK? We normally download these, but ASCII armor keys are maybe a grey area. This one isn't huge at least. Keeping it in-tree would actually have some advantages as often the keyservers do not return stable HTTP results. But then again, this is almost a "binary file"...

Reply:
cc @mgorny

Reply:
> @ulm Is this OK? We normally download these, but ASCII armor keys are maybe a grey area. This one isn't huge at least. Keeping it in-tree would actually have some advantages as often the keyservers do not return stable HTTP results. But then again, this is almost a "binary file"...

It's a text file, so the automatic check won't complain. I don't see a problem since it's short enough.

Reply:
Upstream releases the key with every release (see https://github.com/eduvpn/python-eduvpn-client/releases/download/4.2.0/python-eduvpn-client-4.2.0.tar.xz.asc for example). I didn't found any generic URL to fetch the key from. Embedding the key in the `files`-directory seemed to be the easiest solution. I am open for other suggestions.

Reply:
To me this looks like a URL where the key could be fetched from. What am I missing?

Reply:
If that's the case, then I think it's more reasonable to inline it in the ebuild (via `<<-EOF` thingie) rather than attach as a separate file. That should both save some space and be easier to clean up in the future.

Reply:
fixed